### PR TITLE
refactor(install): eliminate dependency on open-install-library

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,40 @@ $ make test-unit
...
+make recipes
+```
+
+Recipe files are stored in `internal/install/recipes/files`. Once files have been
+fetched, they will be included in future CLI builds. If a particular version of
+the recipe library is desired, the archive download URL can be passed to the make
+target via the `RECIPES_ARCHIVE_URL` option:
+
+```
+make recipes RECIPES_ARCHIVE_URL=https://github.com/newrelic/open-install-library/releases/download/v0.49.28/recipes.zip
```
Comment (Julien4218) [inline README.md:240]: maybe update the version number to our first actual true release of the library. @ctrombley

**Review by Julien4218 — APPROVED** [review]
This looks great.

File: README.md (hunk 1 above)
Comment (ctrombley) [reply README.md]: 👍

File: internal/install/recipes/embedded_recipe_fetcher.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	return out, nil
+}
+
+func (f *EmbeddedRecipeFetcher) getYAMLFiles(path string) (out []string, err error) {
+	return f.getFiles(path, isYAMLFile)
+}
+
+func isYAMLFile(path string) bool {
+	e := filepath.Ext(path)
+	return e == ".yml" || e == ".yaml"
```
Comment (Julien4218) [inline internal/install/recipes/embedded_recipe_fetcher.go:58]: @ctrombley maybe a case insensitive compare, just in case for windows?

**Review by Julien4218 — APPROVED** [review]

File: internal/install/recipes/embedded_recipe_fetcher.go (hunk 2 above)
Comment (ctrombley) [reply internal/install/recipes/embedded_recipe_fetcher.go]: 👍

**Pull request merged**